sdk/computeschedule/arm-computeschedule/review/arm-computeschedule.api.md

```diff
@@ -52,13 +60,56 @@ export interface DeallocateResourceOperationResponse {
     type: string;
 }
 
+// @public
+export interface DeleteResourceOperationResponse {
+    description: string;
+    location: string;
+    results?: ResourceOperation[];
+    type: string;
+}
+
+// @public
+export interface ErrorAdditionalInfo {
+    readonly info?: any;
+    readonly type?: string;
+}
+
+// @public
+export interface ErrorDetail {
+    readonly additionalInfo?: ErrorAdditionalInfo[];
+    readonly code?: string;
+    readonly details?: ErrorDetail[];
+    readonly message?: string;
+    readonly target?: string;
+}
+
+// @public
+export interface ErrorResponse {
+    error?: ErrorDetail;
+}
+
+// @public
+export interface ExecuteCreateRequest {
+    correlationid?: string;
```

Comment:
Could you check if this is expected? It seems inconsistant with correlationId.

Reply:
yes this is expected https://github.com/Azure/azure-rest-api-specs/blob/main/specification/computeschedule/ComputeSchedule.Management/models.tsp#L438